.env.example

```
@@ -0,0 +1 @@
+ZALO_OA_SECRET_KEY=my_zalo_oa_secret_key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)